src/app/search/search.component.css

```diff
@@ -1,58 +1,19 @@
-/* Create the left sidebar */
-#searchContainer {
+/* ----- Blocks ----- */
```

Comment:
Optional: Maybe a better name here?

Reply:
That's what the "B" in "BEM" stands for though. I didn't chose the name. :stuck_out_tongue: